### ISSUE TITLE
[AssetMapper]  Generated `importmap.json` does not take in account `assets.base_urls`

### ISSUE THREAD

**Issue by allan-simon** [symfony/symfony#52561]
### Symfony version(s) affected

6.3.8

### Description

I host my assets on cloudfront on a domain  `assets.example.com` 

during the deployments process I do 

```
    - bin/console asset-map:compile
    - bin/console assets:install
```

my configuration is 

```yaml
framework:
    assets:
        base_urls: 'https://assets.example.com'

```

When deploying 

```html
<link rel="stylesheet" href="{{ asset('css/fonts.css') }}">
```

is correctly transformed into 

```html
<link rel="stylesheet" href="https://assets.rosaly.com/assets/css/fonts-e20b57a8a9c1f3007485f5e84f2b03d4.css">
```

however my importmap.php looks like 
```
return [
      'company_admin_dashboard' => [
          'path' => 'company_admin_dashboard.js',
      ],
```

and when I do

```sh
bin/console asset-map:compile
```

it generates

```json
{
    "imports": {
        "company_admin_dashboard": "/assets/company_admin_dashboard-b1ef2f3239b41a72a0da46d62784a9a8.js",
    }
}
```

So it seems ` bin/console asset-map:compile`  is ignoring my assets configuration 

### How to reproduce

see steps above 

### Possible Solution

I could use an env variable and do something like 

```
return [
      'company_admin_dashboard' => [
          'path' => $_ENV['ASSETS_BASE_URL'].'company_admin_dashboard.js',
      ],
```

which would be empty in local and sets to assets.example.com in production but it feels hackish

**Comment by allan-simon** [comment]
For the moment I manually edit the importmap during CI  

```sh
sed -ie "s@: \"/@: \"$ASSETS_BASE_URL/@"  public/assets/importmap.json
```

(note: I also do during deployment ) 
```
bin/console importmap:require  all mydifferent dependencies  --download
```

so that I don't have to commit dependencies while still being able to have my website not relying on 3rd party for production , which is especially important when you have very strict CSP policies (which everybody having commercial website should)

**Comment by Jean-Beru** [comment]
I reproduced it in 6.3 but 6.4 generates the URL with the correct domain  :
```html
<script type="importmap">
{
    "imports": {
        "app": "https://assets.example.com/assets/app-bbae9f648af22c8bdc294ce35f4a1e4c.js"
    }
}
</script>
```

Since this component is experimental, do we have to make a PR to fix it ? :thinking:

**Comment by Jean-Beru** [comment]
I reproduced it in 6.3 but 6.4 generates the URL with the correct domain  :
```html
<script type="importmap">
{
    "imports": {
        "app": "https://assets.example.com/assets/app-bbae9f648af22c8bdc294ce35f4a1e4c.js"
    }
}
</script>
```

Since this component is experimental, do we have to make a PR to fix it ? :thinking:

**Comment by allan-simon** [comment]
If it's fixed in 6.4 I can wait , I have a workaround in between

**Comment by weaverryan** [comment]
Yup - should all work in 6.4.

Btw, the `asset-map:compile` still generates some compiled files without the `base_urls` prefix. Then, at runtime, the URL is adjusted.

Cheers!
